app/models/katello/concerns/host_managed_extensions.rb

```diff
@@ -32,6 +33,15 @@ def rhsm_organization_label
         self.organization.label
       end
 
+      def info_with_katello
+        info = info_without_katello
+        info['parameters']['kt_env'] = self.lifecycle_environment.try(:label) #deprecated
+        info['parameters']['kt_cv'] = self.content_view.try(:label) #deprecated
```

Comment:
@stbenjam as part of the PR i'm changing it to a 'better' parameter name.  Are those sufficient?  

I added the old ones back as some users were using those even though they were really just used by the old activation keys javascript.